tests/common/snappi_tests/snappi_fixtures.py

```diff
@@ -980,3 +981,39 @@ def pre_configure_dut_interface(duthost, snappi_ports):
         except Exception:
             pytest_assert(False, "Unable to configure ip on the interface {}".format(port['peer_port']))
     return snappi_ports_dut
+
+
+def clear_fabric_counters(duthost):
+    """
+    Clears the fabric counters for the duthost.
+    Args:
+        duthost(obj): dut host object
+    Returns:
+        None
+    """
+    logger.info('Clearing fabric counters for DUT:{}'.format(duthost.hostname))
+    duthost.shell('sonic-clear fabriccountersport \n')
+    time.sleep(1)
+
+
+def check_fabric_counters(duthost):
+    """
+    Check for the fabric counters for the duthosts.
+    Test assert if the value of CRC, and FEC_UNCORRECTABLE.
+    Args:
+        duthost(obj): dut host object
+    Returns:
+        None
+    """
+    raw_out = duthost.shell("show fabric counters port | grep -Ev 'ASIC|---|down'")['stdout']
```

Comment:
Seems this is for broadcom platform only. Should we add a platform check here?

@rraghav-cisco any comments?